assignments/arrays.js

```diff
@@ -63,34 +63,49 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const findId = function(numId) {
+  for(obj in inventory) {
+    if(inventory[obj].id === numId) {
+      return `Car ${inventory[obj].id} is a ${inventory[obj].car_year} ${inventory[obj].car_make} ${inventory[obj].car_model}`
+    }
+  }
+}
+console.log(findId(33));
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+const lastCar = (inventory.length - 1);
+console.log(`The last car is ${inventory[obj].car_make} and ${inventory[obj].car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+function sortCar(a, b) {
+  a = a.toLowerCase()
+  b = b.toLowerCase()
+  return a < b ? -1 : b > a ? 1 : 0
+}
+
+console.log(inventory.map(car => car.car_model).sort(sortCar));
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+const carYears = inventory.map(car => car.car_year);
```

Comment:
you were not suppose to use higher order functions like map in this assignment. Make sure to check the read.me more closely.

However, I like it lol. good job.